### PR TITLE
Added `all` subcommand 

### PR DESCRIPTION
**Pull request by acoomans**
Added an `all` subcommand doing both the downloading of the schema and generating the code

### CODE REVIEW

File: ApolloCodegen/Sources/ApolloCodegen/main.swift [hunk 1]
```diff
@@ -72,6 +72,17 @@ struct SwiftScript: ParsableCommand {
...
     }
+
+    struct DownloadSchemaAndGenerateCode: ParsableCommand {
+        static var configuration = CommandConfiguration(
+            commandName: "all",
+            abstract: "Downloads the schema and generates swift code.")
+
+        func run() throws {
+            try! DownloadSchema().run()
+            try! GenerateCode().run()
```
Comment (designatednerd) [inline ApolloCodegen/Sources/ApolloCodegen/main.swift]: These should be `try` not `try!`, as they are in a throwing function. The function will rethrow with `try`, whereas the whole app will crash with `try!`.

**Review by designatednerd — CHANGES_REQUESTED** [review]
Interesting idea - The main reason we don't presently have this is that our recommended setup is to run codegen on every build. This would add however long it takes to download the schema to the length of time it takes to build your application, which, since it necessarily involves a network operation, is not necessarily desirable. 

However, if you're using a different setup and not generating on every build, this could be quite useful. 

We'll have to weigh the value of this for supporting different setups vs. providing an easy-to-use footgun for people who are following our recommended setup

**Comment by acoomans** [comment]
@designatednerd we want building to be as fast as possible, so we don't add extra script run to it. Instead, the codegen tool is run:
- by the CI if needed (can't remember the details here)
- by an engineer as needed, typically when adding a new graphQL query/mutation or field
In either case, we run the whole thing (download schema and codegen).

File: ApolloCodegen/Sources/ApolloCodegen/main.swift (hunk 1 above)
Comment (acoomans) [reply ApolloCodegen/Sources/ApolloCodegen/main.swift]: True, can change

**Review by designatednerd — APPROVED** [review]
Cool, thanks for clarifying the use case. I'll probably add a note in there about not running it on every build but this is good to merge!

**Comment by acoomans** [comment]
Thanks for merging @designatednerd !

**Pull request merged**